CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to the Openshift Builder
+
+The OpenShift Builder image drives [OpenShift Builds](https://docs.okd.io/latest/dev_guide/builds/index.html),
+leveraging [Buildah](https://github.com/containers/buildah) and [Source-To-Image](https://github.com/openshift/source-to-image)
+to turn source code into images that can be deployed on Kubernetes.
+
+## Getting the code
+
+To get started, [fork](https://help.github.com/articles/fork-a-repo) the [openshift/builder](https://github.com/openshift/builder) repo.
+
+## Developing
+
+### Testing on an OpenShift Cluster
+
+The easiest way to test your changes is to launch an OpenShift 4.x cluster.
+First, go to [try.openshift.com](https://try.openshift.com) to obtain a pull secret and download the installer.
+Follow the instructions to launch a cluster on AWS.
+
+If you want the latest OpenShift installer and `oc` clients, go to the [openshift-release](https://openshift-release.svc.ci.openshift.org/) 
```

Comment:
s/OpenShift installer/'openshift-installer'/
matches the formatting of a similar phrase later in this sentence.

Reply:
Fixed to use the binary name `openshift-install`